src/deploy/functions/tasks.ts

```diff
@@ -291,28 +326,25 @@ export function deleteTopicTask(params: TaskParams, topic: backend.PubSubSpec):
   };
 }
 
-export function schedulerDeploymentHandler(
-  errorHandler: ErrorHandler
-): (task: DeploymentTask) => Promise<any | undefined> {
-  return async (task: DeploymentTask) => {
-    let result;
-    try {
-      result = await task.run();
-      helper.printSuccess(task.fn, task.operationType);
-    } catch (err) {
-      if (err.status === 429) {
-        // Throw quota errors so that throttler retries them.
-        throw err;
-      } else if (err.status !== 404) {
-        // Ignore 404 errors from scheduler calls since they may be deleted out of band.
-        errorHandler.record(
-          "error",
-          backend.functionName(task.fn),
-          task.operationType,
-          err.message || ""
-        );
-      }
-    }
+export const schedulerDeploymentHandler = (errorHandler: ErrorHandler) => async (
```

Comment:
q - why suddenly arrow fn style instead of `export async function schedulerDeploymentHandler...`?